src/components/AdminSettings.vue

```diff
@@ -8,103 +8,98 @@
 			<AssistantIcon class="icon" />
 			{{ t('assistant', 'Nextcloud Assistant') }}
 		</h2>
-		<p><a href="https://docs.nextcloud.com/server/latest/admin_manual/ai/index.html">{{ t('assistant', 'Find more details on how to set up Assistant and recommended backends in the Administration documentation.') }}</a></p>
+		<NcNoteCard type="info">
+			{{ t('assistant', 'Find more details on how to set up Assistant and recommended backends in the Administration documentation.') }}
+			<br>
+			<a href="https://docs.nextcloud.com/server/latest/admin_manual/ai/index.html" class="external link-with-icon" target="_blank">
+				{{ t('assistant', 'Open Administration documentation') }}
```

Comment:
```suggestion
				{{ t('assistant', 'Administration documentation') }}
```

"Open" is redundant.